syntax = "proto3";

package planning;

import "common.proto";

message PaymentTask {
  int32 payment_task_id = 1;
  int32 user_id = 2;
  int32 transaction_id = 3;
  int32 account_id = 4;
  MetaData meta_data = 5;
}


message MetaData {
  PlanType preferred_plan_type =1;
  double preferred_timeline=2;
  PaymentFrequency preferred_payment_freq=3;
}